pkg/payload/precondition/clusterversion/upgradeable.go

```diff
@@ -50,6 +50,18 @@ func (pf *Upgradeable) Run(ctx context.Context, releaseContext precondition.Rele
 		return nil
 	}
 
+	// if ClusterStatusConditionType = "Progressing" we should not upgrade
+	if c := resourcemerge.FindOperatorStatusCondition(cv.Status.Conditions, configv1.OperatorProgressing); c != nil {
+		if c.Status == configv1.ConditionTrue {
+			return &precondition.Error{
+				Nested:  err,
+				Reason:  "UpgradePreconditionCheckFailed",
+				Message: "An upgrade already in progress",
```

Comment:
nit: Maybe "Cannot change the update target before the current update completes." or some such?